docs/sphinx/index.rst

```diff
@@ -1,140 +1,22 @@
 Python Elasticsearch Client
 ===========================
 
```

Comment:
Should we link to the main docs?

Reply:
Yes, we should add links to the guide in all the reference pages where there is a reasonable page to link to. I'm looking into this.